apps/ejabberd/src/mod_muc_db.erl

```diff
@@ -0,0 +1,12 @@
+-module(mod_muc_db).
+
+-callback init(ejabberd:server(), list()) -> ok.
+-callback store_room(ejabberd:server(), ejabberd:server(), mod_muc:room(), list())
+            -> {'aborted',_} | {'atomic',_}.
+-callback restore_room(ejabberd:server(), ejabberd:server(), mod_muc:room())
+                                    -> 'error' | 'undefined' | [any()].
+-callback forget_room(ejabberd:server(), ejabberd:server(), mod_muc:room()) -> 'ok'.
```

Comment:
In ODBC backend the last call in this function is `mongoose_rdbms:sql_transaction` and its result is not simple `ok`.